sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -101,7 +101,31 @@ gtest_discover_tests(azure-core-test
      NO_PRETTY_TYPES
      NO_PRETTY_VALUES)
 
+if(BUILD_TRANSPORT_CURL)
+  ################## Azure Libcurl Core Test #################################
+  # Creating one exe file alone for this test since it requires the full control over the connection pool.
+  # This test will check that end-user can call `curl_global_cleanup()` with active handlers in the connection pool
+  # without getting errors.
+  add_executable (
+    azure-core-libcurl-test
+    azure_libcurl_core_main.cpp
+  )
+
+  if (MSVC)
+    target_compile_options(azure-core-libcurl-test PUBLIC /wd26495 /wd26812)
```

Comment:
nit: Can you add a comment for what each of these warnings are that are being ignored?